tests/javascripts/templateFolderForm.test.js

```diff
@@ -309,6 +309,36 @@ describe('TemplateFolderForm', () => {
 
   });
 
+  describe("Click 'New template' for single channel service", () => {
+    test("should redirect to new template page", () => {
+      setFixtures(hierarchy, "data-channel='sms' 'data-service='123'")
```

Comment:
There’s an extra quotation mark here:
```suggestion
      setFixtures(hierarchy, "data-channel='sms' data-service='123'")
```